skimage/transform/tests/test_warps.py

```diff
@@ -228,6 +228,17 @@ def test_swirl():
         swirled = tf.swirl(image, strength=10, **swirl_params)
         unswirled = tf.swirl(swirled, strength=-10, **swirl_params)
 
+
+    assert np.mean(np.abs(image - unswirled)) < 0.01
+  
+
+    swirl_params.pop('mode')
+
+    with expected_warnings(['Bi-quadratic.*bug', 'default']):
+        swirled = tf.swirl(image, strength=10, **swirl_params)
+        unswirled = tf.swirl(swirled, strength=-10, **swirl_params)
```

Comment:
@LachlanD unfortunately, the border errors that motivate this whole change come to bite you in this case, and the difference between the images becomes greater than 0.01. One way to fix this is by changing the comparison to use a slightly cropped version of the images:

```python
assert np.mean(np.abs(unswirled[1:-1, 1:-1] - image[1:-1, 1:-1])) < 0.01
```

Also, you need to reduce the whitespace in this function — by convention, there should never be more than one consecutive blank line within a Python function.